[quant][graphmode][fx] Add warning for unsupported case

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #45920 [quant][fx][graphmode][api] Change API for custom module
* #45919 [quant][fx][graphmode] Run symbolic_trace in quantization
* #45885 [quant][fx][graphmode] Add support for sigmoid
* #45884 [quant][eagermode] Add support for sigmoid module
* #45883 [quant] Add quantized Sigmoid module
* #45882 [quant] Add quantized::sigmoid that take output_scale/output_zero_point as input
* **#45714 [quant][graphmode][fx] Add warning for unsupported case**

Summary:
Hit the problem when writing a test like following:
```
class M(...):
      def forward(self, x):
          x = x.some_op()
          return x
```
we need to know the scope of `x` to figure out the qconfig for `x`

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D24069959](https://our.internmc.facebook.com/intern/diff/D24069959)

jerryzh168 committed:
[quant][graphmode][fx] Add warning for unsupported case

Summary:
Hit the problem when writing a test like following:
```
class M(...):
      def forward(self, x):
          x = x.some_op()
          return x
```
we need to know the scope of `x` to figure out the qconfig for `x`

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fx] Add warning for unsupported case"


Summary:
Hit the problem when writing a test like following:
```
class M(...):
      def forward(self, x):
          x = x.some_op()
          return x
```
we need to know the scope of `x` to figure out the qconfig for `x`

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fx] Add warning for unsupported case"


Summary:
Hit the problem when writing a test like following:
```
class M(...):
      def forward(self, x):
          x = x.some_op()
          return x
```
we need to know the scope of `x` to figure out the qconfig for `x`

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D24069959](https://our.internmc.facebook.com/intern/diff/D24069959)

[ghstack-poisoned]

z-a-f reviewed (approved):
Is there something like `warn_once` or `torch.warn` for filtering?

jerryzh168 committed:
Update on "[quant][graphmode][fx] Add warning for unsupported case"


Summary:
Hit the problem when writing a test like following:
```
class M(...):
      def forward(self, x):
          x = x.some_op()
          return x
```
we need to know the scope of `x` to figure out the qconfig for `x`

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D24069959](https://our.internmc.facebook.com/intern/diff/D24069959)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fx] Add warning for unsupported case"


Summary:
Hit the problem when writing a test like following:
```
class M(...):
      def forward(self, x):
          x = x.some_op()
          return x
```
we need to know the scope of `x` to figure out the qconfig for `x`

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D24069959](https://our.internmc.facebook.com/intern/diff/D24069959)

[ghstack-poisoned]
